alibi_detect/od/backends/torch/knn.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import torch
+
+
+class KNNTorch:
+    def score(X, x_ref, k, kernel=None):
+        ensemble = isinstance(k, (np.ndarray, list, tuple))
```

Comment:
Would like to see `list` as option for `k` but currently the top level method only allows types `Union[int, np.ndarray]`.

Reply:
Also already have a `self.ensemble` attribute so no need to recompute.